RInAppMessaging/Classes/Views/BaseView.swift

```diff
@@ -83,6 +83,9 @@ internal extension BaseView {
         NSLayoutConstraint.activate(constraintsForParent(parentView))
 
         parentView.layoutIfNeeded()
-        animateOnShow()
+        parentView.isUserInteractionEnabled = false
```

Comment:
I only need a confirmation here. The `parentView` is one of the IAM main views right? (like modal or banner?)

Reply:
no, it's one of host app (root) views. It's one of keyWindow's subviews